core/src/main/scala/chisel3/experimental/dataview/DataView.scala

```diff
@@ -170,7 +170,7 @@ object DataView {
     )
   }
 
-  /** Provides implementations of [[DataView]] for [[Tuple2]] to [[HWTuple2]] */
+  /** Provides implementations of [[DataView]] for `Tuple2`  to [[HWTuple2]] */
```

Comment:
```suggestion
  /** Provides implementations of [[DataView]] for [[scala.Tuple2]]]  to [[HWTuple2]] */
```
Can't you do the same thing here you did in `DataProduct.scala`?